server/handlers.go

```diff
@@ -54,6 +54,10 @@ import (
 	textTemplate "text/template"
 	"time"
 
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/ProtonMail/go-crypto/openpgp/armor"
+	"github.com/ProtonMail/go-crypto/openpgp/packet"
+	"github.com/ProtonMail/gopenpgp/v2/constants"
```

Comment:
NIT: this is not properly sorted with the below list.

Reply:
it seems that uppercase is order before lower one